packages/experimental/Button/src/Button.types.ts

```diff
@@ -74,17 +74,17 @@ export interface ButtonTokens extends ButtonCoreTokens {
 }
 
 export interface ButtonCorePropsWithInnerRef extends Omit<IWithPressableOptions<ViewProps>, 'onPress'> {
-  /*
-   * Text to show on the Button.
-   * Only used in FAB
-   */
-  content?: string;
-
   /*
    * Source URL or name of the icon to show on the Button.
    */
   icon?: IconSourcesType;
 
+  /**
+   * Button contains only icon, there's no text content
+   * Must be set for button to style correctly when button has not content.
+   */
+  iconOnly?: boolean;
```

Comment:
So this is mutually exclusive to having text for the Button? If a user has both text + iconOnly=true, what happens?

Reply:
Currently the result would be the styling would look a little weird.

We have a few bugs around mutually exclusive props that I'm hoping to clean up today or tomorrow.

Reply:
Handled in #1283